manifests/charts/dataplane/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: dataplane
+description: Helm umbrella chart for Istio dataplane/singleton components
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+kubeVersion: ">= 1.23.0-0"
+keywords:
+  - istio-cni
+  - ztunnel
+  - istio
+  - dataplane
+  - mesh
+sources:
+  - https://github.com/istio/istio
+icon: https://istio.io/latest/favicons/android-192x192.png
+
+# These will be stamped out to real versions during release publish
+dependencies:
+  - name: base
```

Comment:
I do agree with the proposed UX - but not sure this is the right implementation.

We can achieve the same with adding an option to ztunnel chart ( for example ) that adds the deamon set for cni and whatever is in base - leaving base/cni alone for istio-classic. 

UX will be what you propose - one chart for the data plane.

Reply:
Yeah the UX would come to the same - I prefer this composition approach tho because it doesn't change or alter the underlying charts in any way, and still permits each underlying chart to be installed separately, just as we do today - so it's purely additive and doesn't alter any existing charts we already publish.

I consider it something of a compromise, I guess.